test/e2e/tests/test_adopt_endpoint.py

```diff
@@ -0,0 +1,243 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the
+# License is located at
+#
+# 	 http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+"""Integration tests for the SageMaker Endpoint API.
+"""
+
+import boto3
+import pytest
+import logging
+import time
+from typing import Dict
+
+from acktest.aws import s3
+from acktest.resources import random_suffix_name
+from acktest.k8s import resource as k8s
+
+from e2e import (
+    service_marker,
+    CRD_GROUP,
+    CRD_VERSION,
+    ENDPOINT_CONFIG_RESOURCE_PLURAL,
```

Comment:
you shouldn't have to import these anymore from e2e since they moved to the config.